app/frontend/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js --host 0.0.0.0",
     "start": "npm run dev",
-    "unit": "jest --config test/unit/jest.conf.js --coverage",
+    "unit": "jest -u --config test/unit/jest.conf.js --coverage",
```

Comment:
Would the snapshots have any usefulness after adding this flag?  I think if not, it would make more sense to disable the snapshot tests for components that are in development (until they are stable)

Reply:
@byshine yes we should be only calling `-u` manually to update snapshots. Good catch @stephenhillier

Reply:
@countable @stephenhillier i agree with you stephen, I just forgot to change it back.